docs/website/root/manual/develop/nodes/mithril-client.md

```diff
@@ -227,27 +227,45 @@ Commands:
 
 Options:
       --run-mode <RUN_MODE>
-          Run Mode [env: RUN_MODE=] [default: dev]
+          Run Mode
+
+          [env: RUN_MODE=]
+          [default: dev]
+
   -v, --verbose...
           Verbosity level (-v=warning, -vv=info, -vvv=debug, -vvvv=trace)
+
       --config-directory <CONFIG_DIRECTORY>
-          Directory where configuration file is located [default: ./config]
+          Directory where configuration file is located
+
+          [default: ./config]
+
       --aggregator-endpoint <AGGREGATOR_ENDPOINT>
```

Comment:
` [env: AGGREGATOR_ENDPOINT=]` disappeared, is that normal ?
I found that strange since env source is still available in the cli argument and it's still here for `--run-mode`.